src/types/TransactionReceipt.ts

```diff
@@ -0,0 +1,23 @@
+import {ExchangeRateSet, exchangeRateSetToSdk} from "./ExchangeRateSet";
+import {TransactionReceipt as ProtoTransactionReceipt} from "../generated/TransactionReceipt_pb";
+import {AccountId, accountIdToSdk} from "./AccountId";
+import {ContractId, contractIdToSdk} from "./ContractId";
+import {FileId, fileIdToSdk} from "./FileId";
+
+export type TransactionReceipt = {
```

Comment:
We can probably stand to change `Transaction.executeForReceipt()` to return this type so people don't have to use the protobufs.